Okta.Xamarin/Okta.Xamarin/OktaStateManager.cs

```diff
@@ -80,6 +87,30 @@ public bool IsAuthenticated
             }
         }
 
+        /// <summary>
+        /// Gets or sets the last response received from the API.
+        /// </summary>
+        public HttpResponseMessage LastApiResponse { get; set; }
```

Comment:
Curious, what's the purpose of this property?

Reply:
This is mostly for debugging.  I'll add that to the comment.